model/cat/indices/cat_indicies/structures.smithy

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
+// 
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+// 
+
+namespace OpenSearch
+
+structure GetCatIndicesInput {
+    // Common options to be removed by mixins start
+
+    @httpQuery("bytes")
+    bytes: Byte,
+
+    @httpQuery("expand_wildcards")
+    expand_wildcards: ExpandWildcards,
+
+    @httpQuery("health")
+    health: HealthStatus,
+
+    @httpQuery("include_unloaded_segments")
+    include_unloaded_segments: Boolean,
+
+    @httpQuery("pri")
+    pri: Boolean,
+
+    @httpQuery("master_timeout")
+    master_timeout: Time,
+
+    @httpQuery("timeout")
+    timeout: Time,
+    // Common options end
+
+}
+
+structure GetCatIndicesOutput {
+
+   @required
+   body:Document
```

Comment:
cat gives us a json only when a URL parameter is passed. Can we replicate that here?

Reply:
if we are passing index in the URL parameter we can do that but for generic one (Get /_cat/indicies) need to check.

Reply:
Cat indexes passes table by default, since we are working on API we should hardcode format=JSON

Reply:
Will work on it